PrivateHeaders/APPX/ZIP.h

```diff
@@ -327,6 +328,11 @@ namespace appx {
            << "xmlns=\"http://schemas.microsoft.com/appx/2010/blockmap\" "
            << "HashMethod=\"http://www.w3.org/2001/04/xmlenc#sha256\">";
         for (const ZIPFileEntry &entry : otherEntries) {
+            const std::string suffix = ".appx";
+            if (suffix.size() < entry.fileName.size() &&
+                    std::equal(suffix.rbegin(), suffix.rend(), entry.fileName.rbegin())) {
+                continue;
+            }
```

Comment:
Why are we skipping .appx files? If it's for APPX bundles, shouldn't we only skip if we are making a bundle?